lua/caramba/utils.lua

```
@@ -51,6 +51,58 @@ function M.show_result_window(content, title)
+    col = col,
+    width = width,
+    height = height,
+    style = "minimal",
+    border = "rounded",
+    title = title,
+    title_pos = "center",
+  })
+
+  vim.api.nvim_buf_set_option(buf, "filetype", "markdown")
+  vim.api.nvim_buf_set_option(buf, "bufhidden", "wipe")
+  vim.api.nvim_buf_set_option(buf, "modifiable", true)
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)